tests/subfolder/.another.env

```diff
@@ -0,0 +1 @@
+CUSTOM_STRING=bar
```

Comment:
Is this file actually used anywhere?

Reply:
No but it was required to check in the subfolder for one of the tests.